core/adapters/sqldatawarehouse.ts

```diff
@@ -0,0 +1,109 @@
+import { dataform } from "@dataform/protos";
+import { Task, Tasks } from "../tasks";
+import { Adapter } from "./base";
+import { IAdapter } from "./index";
+
+export class SQLDataWarehouseAdapter extends Adapter implements IAdapter {
+  private project: dataform.IProjectConfig;
+
+  constructor(project: dataform.IProjectConfig) {
+    super();
+    this.project = project;
+  }
+
+  private createOrReplace(table: dataform.ITable) {
+    if (table.type === "view") {
+      return (
+        Tasks.create()
+        // Drop the view in case we are changing the number of column(s) (or their types).
```

Comment:
do you know if this is required for SQL datawarehouse?

Reply:
yes, it is